### ISSUE TITLE
Failure deploying from Bitbucket git to Linux Web App

### ISSUE THREAD

**Issue by NVV-WebManager** [microsoft/Oryx#1897]
# The Issue
I'm unable to deploy via git to an Azure Web App provisioned for Linux.  Most of the deployment processes successfully, but fails at the very end with status: "Deployment Failed. deployer = Bitbucket deploymentPath ="

Looking further into the oryx logs, the last point of entry is:

```
@php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1\n/bin/bash -c "oryx build /home/site/repository -o /home/site/wwwroot --platform php --platform-version 8.1 -p virtualenv_name= --log-file /tmp/build-debug.log  -i /tmp/8db1128e91afb0c | tee /tmp/oryx-build.log ; exit $PIPESTATUS "
```
While I was able to manually deploy the site via ssh, I'd like to resolve the issues with automatic deployment. However, I'm not clear on what is actually causing the issue. There's some reference higher in the logs about missing the pdo_sqlsrv driver, but processing does continue past that point and the driver is available on the site after deployment. 

# Oryx Log File
Entire log from the deployment center is attached.
[oryx.log](https://github.com/microsoft/Oryx/files/10798574/oryx.log)

# Expected Result
Pushing a commit to my configured git branch would successfully process the Oryx build and deploy site updates.

# Project Specs
Laravel 9 application running on PHP 8.1, connecting to an MS SQL Server database via pdo_sqlsrv driver.

**Comment by william-msft** [comment]
Hi @NVV-WebManager thank you for reaching out. Would you mind re-deploying? I wonder if this could be related to a connectivity related issue?

**Comment by NVV-WebManager** [comment]
> Hi @NVV-WebManager thank you for reaching out. Would you mind re-deploying? I wonder if this could be related to a connectivity related issue?

Hi William.  It is most definitely not a connectivity issue.  I have already tried half a dozen times over several days, with the same result in every case.  

However, I had a very small one page application that I was able to deploy yesterday via git to a linux web app configured in exactly the same way as the failing one.  One major difference between the projects is that the successful one (besides being far less complicated and smaller a codebase) has no database connection.

**Comment by NVV-WebManager** [comment]
The issue appears to have been specific to the missing driver error posted in the logs.  I noticed that the code/package triggering that error was a dev dependency that I didn't actually want or need in my deployed environment, so I removed it entirely from the project.  The code then deployed and launched as expected. 

I'm still not clear on why the Oryx script was freaking out about that package/driver when the same was not true of my manual deployment via ssh.  That still seems like some kind of bug, but my immediate has been resolved.
